### ISSUE TITLE
NTP Image shows only after fresh install

### ISSUE THREAD

**Issue by Brandon-T** [brave/brave-ios#2288]
### Description: 
- Fresh install, NTP image shows.
- After a few hours or minutes or the refresh time has passed (while the app is still in memory and open), the image does NOT update!
- It always shows the old image no matter what.

### Steps to Reproduce 
  1.
  2.
  3.

**Actual result:** 

**Expected result:**

**Reproduces how often:** [Easily reproduced, Intermittent Issue]

**Brave Version:** 

**Device details:** 

**Website problems only:**
- did you check with Brave Shields down?
- did you check in Safari/Firefox (WkWebView-based browsers)?

**Comment by iccub** [comment]
Duplicate of https://github.com/brave/brave-ios/issues/2287
